package/yast2-security.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-security
-Version:        4.4.9
+Version:        4.4.10
```

Comment:
do not forget to bump also schema spec version as you modify schema, so it has to be regenerated

Reply:
Yep, it is here https://github.com/yast/yast-schema/pull/127